wcfsetup/install/files/lib/data/user/online/UserOnline.class.php

```diff
@@ -215,9 +215,14 @@ public function getBrowser() {
 		}
 		
 		// edge
-		if (preg_match('~edge/(\d{2}\.\d+)~i', $this->userAgent, $match)) {
+		if (preg_match('~edge/(\d{2}\.\d+)|edg/([\d\.]+)~i', $this->userAgent, $match)) {
 			return 'Microsoft Edge '.$match[1];
 		}
+
+		// edge mobile (new)
+		if (preg_match('~edga/([\d\.]+)~i', $this->userAgent, $match)) {
+			return 'Microsoft Edge Mobile '.(isset($match[2]) ? $match[2] : $match[1]);
```

Comment:
Likewise there won't  ever be a `$match[2]` here. Did you mix up the return statements?